docs/source/conf.py

```diff
@@ -0,0 +1,91 @@
+# conf.py
+# Configuration file for the Sphinx documentation builder.
+project = 'cookiecutter-python'
+copyright = "2025, Neuroinformatics Unit"
+author = 'NIU'
+release = "1.0.0"
+
+# -- General configuration ---------------------------------------------------
+extensions = [
+    "myst_parser",
+    "sphinx_design",
+    "sphinx.ext.autodoc",
+    "sphinx.ext.napoleon",
+    "sphinx.ext.githubpages",
+    "sphinx_autodoc_typehints",
+    "sphinx.ext.autosummary",
+    "sphinx.ext.viewcode",
+    "sphinx.ext.intersphinx",
+    "sphinx_sitemap",
+    "nbsphinx",
+]
+
+# Configure MyST Parser
+myst_enable_extensions = [
+    "colon_fence",
+    "deflist",
+    "fieldlist",
+    "html_admonition",
+    "html_image",
+    "linkify",
+    "replacements",
+    "smartquotes",
+    "substitution",
+    "tasklist",
+    "attrs_inline"
+]
+
+myst_heading_anchors = 4
+
+source_suffix = {
+    '.md': 'markdown',
+    '.rst': 'restructuredtext'
+}
+
+templates_path = ['_templates']
+exclude_patterns = [
+    "**.ipynb_checkpoints",
+    "**/includes/**",
+]
+
+# -- HTML output options -----------------------------------------------------
+html_theme = 'pydata_sphinx_theme'
+html_logo = "_static/logo.png"
+html_static_path = ["_static"]
+html_title = "Python Cookiecutter"
+html_favicon = "_static/favicon.ico"
+html_show_sourcelink = False
+
+html_theme_options = {
+    "navbar_start": ["navbar-logo"],
+    "navbar_center": ["navbar.html"],
+    "navbar_end": ["theme-switcher", "navbar-icon-links"],
+    "icon_links": [
+        {
+            "name": "GitHub",
+            "url": "https://github.com/neuroinformatics-unit/python-cookiecutter",
+            "icon": "fa-brands fa-github",
+            "type": "fontawesome",
```

Comment:
This doesn't seem to be showing up in the built website for some reason. Would it also be possible to add a link to the relevant zulip channel, similar to how `movement` does it:

https://github.com/neuroinformatics-unit/movement/blob/0c1110329da790e5ce5813494980dd71f0f219ed/docs/source/conf.py#L126-L152